src/main/java/net/fabricmc/loom/configuration/ide/RunConfig.java

```diff
@@ -149,6 +149,7 @@ private static void populate(Project project, LoomGradleExtension extension, Run
 		}
 	}
 
+	@Deprecated // Replaced by runConfig(project, settings)
```

Comment:
This isnt an API so feel free to remove it fully.

Reply:
Ok, `RunClientTask` and `RunServerTask` still use them but I'll redirect these tasks to `runConfig(...)`